test/src/e2e_vm_tests/test_programs/should_pass/require_contract_deployment/get_storage_key_caller/src/main.sw

```diff
@@ -26,5 +26,11 @@ fn main() -> u64 {
 
     assert(f3 != f4);
 
+    let(cf1, cf2, cf3, cf4) = caller.from_callers();
+    assert(f1 == cf1);
+    assert(f2 == cf2);
+    assert(f3 == cf3);
+    assert(f4 == cf4);
```

Comment:
The names could certainly be a little more descriptive for anyone wondering why they should be equal